code/modules/research/designs/autolathe_desings/autolathe_designs_electronics.dm

```diff
@@ -74,3 +74,11 @@
 	materials = list(/datum/material/glass = 20)
 	build_path = /obj/item/stock_parts/cell/emergency_light
 	category = list("initial", "Electronics")
+
+/datum/design/electrochromatic_control
+	name = "Electrochromatic Control Circuit"
+	id = "electrochromatic_control_circuit"
+	build_type = AUTOLATHE
+	materials = list(/datum/material/iron = 100, /datum/material/glass = 100)
```

Comment:
maybe tweak this to be a bit more metal wise.

Reply:
y

Reply:
It takes a full metal sheet to make, theirs no way that 1/20th of a metal sheet can cover a full glass sheet, even rods are 1/4th the mats to upgrade a single tile

Reply:
spraycans are a way cheaper way of making glass not seethrough, and are technically "toggleable" via paint remover. These are only a little more expensive and take setup to make.

Reply:
what lolman said, shouldn't take that much to do this.